ibm_catalog.json

```diff
@@ -576,6 +757,10 @@
               {
                 "title": "Deploys the IBM Cloud Monitoring agent on an existing cluster on a secure landing zone.",
                 "description": "Deploys the IBM Cloud Monitoring agent on an existing Red Hat OpenShift cluster on a secure landing zone."
+              },
+              {
+                "title": "Deploys the IBM Logs agent on an existing cluster.",
+                "description": "Deploys the IBM Logs agent on an existing Red Hat OpenShift cluster on a secure landing zone."
```

Comment:
Hmm, I thought a left a comment on this already - remove landing zone from here and the other features that mention it to. This DA can be deployed on any cluster, not just landing zone